integr8ly/integreatly-operator
# MGDAPI-6066 bump kc

# Issue link
https://issues.redhat.com/browse/MGDAPI-6066

# What
KC bump to latest version

# Verification steps
- Provision cluster
- run `make cluster/prepare/local`
- create catalog source with 1.38.0:
```
cat << EOF | oc create -f -
apiVersion: operators.coreos.com/v1alpha1
kind: CatalogSource
metadata:
  name: rhoam-operators
  namespace: openshift-marketplace
spec:
  sourceType: grpc
  image: quay.io/mstoklus/managed-api-service-index:1.38.0
EOF
```
- create RHMI CR:
```
IN_PROW=false USE_CLUSTER_STORAGE=true make deploy/integreatly-rhmi-cr.yml
```
- Install RHOAM via operator hub
- Once installed, bump the catalog source yaml image to 1.39.0 (same org - just flip version)
- Approve the RHOAM upgrade
- Wait for RHOAM upgrade to fully complete and confirm both sso versions are at 7.6.5-opr-004
- Uninstall RHOAM

Also confirm fresh install works based on passing e2e on this PR.
